LeaderboardBackend.Jobs/Jobs/ScaffoldUser.cs

```diff
@@ -0,0 +1,28 @@
+using System.ComponentModel.DataAnnotations;
+using LeaderboardBackend.Jobs.Core;
+using LeaderboardBackend.Models.Entities;
+
+
+namespace LeaderboardBackend.Jobs.Jobs;
+
+internal class ScaffoldUser : IJob
+{
+	private ApplicationContext Context;
+
+	public ScaffoldUser(ApplicationContext context)
+	{
+		Context = context;
+	}
+
+	public string CommandName => "user";
+	public string Description => "Create a new user.";
+	public override string ToString() => $"{CommandName}: {Description}";
```

Comment:
Could we add these fields to the interface instead? Or would (for lack of a better term) adding `ToString` mean that `IJob` would have to be an abstract class? Combination request and query about basic C# here

Reply:
These fields are on the interface

Reply:
Changed the Run() to implement implicitly so they are all consistent